extensions/nns/src/install_nns.rs

```diff
@@ -100,12 +169,50 @@ pub async fn install_nns(
             .with_context(|| format!("Could not parse url for {canister_name} wasm: {wasm_url}"))?;
         download(&parsed_wasm_url, &local_wasm_path).await?;
         let specified_id = Principal::from_text(canister_id)?;
+        let arg = if *canister_id == NNS_DAPP.canister_id {
+            let nns_dapp_metadata = vec![
+                ("API_HOST".to_string(), nns_url.to_string()),
+                ("CKETH_INDEX_CANISTER_ID".to_string(), ICRC1_INDEX.canister_id.to_string()),
+                ("CKETH_LEDGER_CANISTER_ID".to_string(), ICRC1_LEDGER.canister_id.to_string()),
+                ("CYCLES_MINTING_CANISTER_ID".to_string(), "rkp4c-7iaaa-aaaaa-aaaca-cai".to_string()),
+                ("DFX_NETWORK".to_string(), "local".to_string()),
+                ("FEATURE_FLAGS".to_string(), "{\"ENABLE_CKBTC\":false,\"ENABLE_CKTESTBTC\":false,\"ENABLE_HIDE_ZERO_BALANCE\":true,\"ENABLE_VOTING_INDICATION\":true}".to_string()),
+                ("FETCH_ROOT_KEY".to_string(), "true".to_string()),
+                ("GOVERNANCE_CANISTER_ID".to_string(), "rrkah-fqaaa-aaaaa-aaaaq-cai".to_string()),
+                ("HOST".to_string(), nns_url.to_string()),
+                ("IDENTITY_SERVICE_URL".to_string(), format!("http://{}.localhost:8080", INTERNET_IDENTITY.canister_id)),
+                ("INDEX_CANISTER_ID".to_string(), ICP_INDEX.canister_id.to_string()),
+                ("LEDGER_CANISTER_ID".to_string(), "ryjl3-tyaaa-aaaaa-aaaba-cai".to_string()),
+                ("OWN_CANISTER_ID".to_string(), NNS_DAPP.canister_id.to_string()),
+                ("ROBOTS".to_string(), "<meta name=\"robots\" content=\"noindex, nofollow\" />".to_string()),
+                ("SNS_AGGREGATOR_URL".to_string(), format!("http://{}.localhost:8080", SNS_AGGREGATOR.canister_id)),
+                ("STATIC_HOST".to_string(), nns_url.to_string()),
+                ("TVL_CANISTER_ID".to_string(), "".to_string()),
+                ("WASM_CANISTER_ID".to_string(), "qaa6y-5yaaa-aaaaa-aaafa-cai".to_string())
+            ];
+            let nns_dapp_init_args = Some(CanisterArguments {
+                args: nns_dapp_metadata,
+                schema: Some(SchemaLabel::AccountsInStableMemory),
+            });
+            Some(Encode!(&nns_dapp_init_args).unwrap())
+        } else if *canister_id == SNS_AGGREGATOR.canister_id {
+            Some(
+                Encode!(&Some(Config {
+                    update_interval_ms: 1_000,
+                    fast_interval_ms: 100,
```

Comment:
These parameters are crazy fast and cause a lot of unnecessary updates and a lot of spam in the local replica logs.

We typically use
```
update_interval_ms: 120_000,
fast_interval_ms: 10_000,
```
and even that produces a lot of output.

I've already made a change to stop producing all this output (which will only take effect after the next release) but even then it's unnecessary to make the aggregator do so much work.

Reply:
With a long update interval, you need to wait for 2 mins until your SNS appears on the dashboard, no?

Reply:
In snsdemo we temporarily change the configuration until all the SNSes are loaded:
https://github.com/dfinity/snsdemo/blob/main/bin/dfx-sns-aggregator-wait

I'm working around these parameters in snsdemo by reinstalling the aggregator so this doesn't need to be fixed for us.
But it definitely doesn't seem ideal like this.

Reply:
> But it definitely doesn't seem ideal like this.

Why? To reduce the amount of logs? If so, then ideally the log level is configurable in the canister init args and the nns extension of dfx could set an appropriate log level.

> In snsdemo we temporarily change the configuration

What's the alternative for the dfx extension? Creating a dfx extension command to reconfigure the aggregator and tell people to run it before and after their SNS deployment?